README.md

```diff
@@ -1,75 +1,85 @@
+- [Service Nodes Guides](#service-nodes-guides)
 - [Docker](#docker)
+- [Testnet](#testnet)
+- [Software Integrity and Authenticity](#software-integrity-and-authenticity)
 - [Contributing](#contributing)
 - [License](#license)
 
-## Background
+### Reference
 
-- [Nimiq Proof-of-Stake Portal](https://www.nimiq.com/albatross/): General information and high level details of the Nimiq Proof-of-Stake blockchain
-- [Nimiq Albatross White Paper](https://arxiv.org/abs/1903.01589): White paper describing the consensus algorithm used in Nimiq Proof-of-Stake
-- [Nimiq Proof-of-Stake migration process](https://www.nimiq.com/blog/nimiq-20-albatross-hard-fork-preparations/): Migration process to Nimiq Proof-of-Stake
-- [Nimiq 1.0 Developer Reference](https://nimiq-network.github.io/developer-reference/): Details of the protocol architecture.
-- [Nimiq 1.0 JavaScript implementation](https://github.com/nimiq/core-js/): Nimiq 1.0 implementation
+- [Nimiq Proof-of-Stake Portal](https://www.nimiq.com/albatross/): General information and high level details of the Nimiq Proof-of-Stake blockchain.
+- [Nimiq Albatross White Paper](https://arxiv.org/abs/1903.01589): White paper describing the consensus algorithm used in Nimiq Proof-of-Stake.
+- [Nimiq Developer Center](https://www.nimiq.com/developers/): The place for all the developer documentation and protocol design implementation.
+- [JSON-RPC Specification](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/): Documentation for interacting with the network using JSON-RPC.
+- [Nimiq Proof-of-Stake Migration Technicalities](https://www.nimiq.com/developers/migration/migration-technical-details): Migration process to Nimiq Proof-of-Stake.
+- [Migration for Integrators](https://www.nimiq.com/developers/migration/migration-integrators): A guide for those who want a more in depth overview of the transitioning process from Proof-of-Work to Proof-of-Stake.
+- [Blockchain Explorer](https://www.nimiqhub.com/): NimiqHub allows you to switch from Testnet to Mainnet.
 
-## System requirements
+## Hardware Requirements per Node Type
 
-- 64-bit computing architecture.
-- 2 or more CPU cores
-- 2GB of available RAM memory
-- 5GB of free disk space, an SSD is highly recommended (100GB or more is required for history nodes)
-- File systems with sparse file support.
-- It is highly recommended to run a clock synchronization protocol such as NTP. This
-  is needed to properly accept blocks according to the timestamp and it is especially
-  important for validators in order to produce blocks in the expected timestamps.
+| PoS Node Type | Memory | CPU | Storage | Network | Syncing Time |
+| --- | --- | --- | --- | --- | --- |
+| **History** | Minimum 16GB RAM (higher recommended) | Minimum 4 vCPUs, 8 recommended | Starts from a few gigabytes, grows linearly with blockchain size | High-speed, reliable internet connection; Good I/O performance (SSDs required) | Sync time increases over the life of the blockchain |
```

Comment:
For storage I'd specify a recommended value. I'd say: 1TB. And maybe add that this space is not going to be consumed from the start but since the history is expected to grow over time, it will need this additional storage over time (which I think is what you meant but it's not clear to me).

Reply:
And maybe also 2TB when enabling indexing